docs/dev/build_riscv64.md

```diff
@@ -0,0 +1,184 @@
+- [Banana Pi BPI-F3](https://www.banana-pi.org/en/banana-pi-sbcs/175.html) with RVV 1.0
+
+
+## Software requirements
+
+- [CMake](https://cmake.org/download/) 3.13 or higher
+- GCC 7.5 or higher (for non-RVV) / [xuantie-gnu-toolchain](https://github.com/XUANTIE-RV/xuantie-gnu-toolchain) (for RVV)
+- Python 3.10 for OpenVINO Runtime Python API
+
+## How to build
+### Build with RVV
+0. Prerequisite - build `xuantie-gnu-toolchain` and `qemu`:
+   ```sh
+   git clone https://github.com/XUANTIE-RV/xuantie-gnu-toolchain.git
+   cd xuantie-gnu-toolchain
+   ./configure --prefix=<xuantie_install_path>
+   make linux build-qemu -j$(nproc)
+   ```
+   > **NOTE**: The `build-qemu` target is optional, as it is used to build the `qemu` simulator. However, it is recommended to build the `qemu` simulator, since it is much more convenient to validate the software on your host than on your devices.
+
+1. Clone OpenVINO repository and init submodules:
+   ```sh
+   git clone https://github.com/openvinotoolkit/openvino.git
+   cd openvino
+   git submodule update --init --recursive
+   ```
+
+2. Install build dependencies using the `install_build_dependencies.sh` script in the
+   project root folder.
+   ```sh
+   sudo ./install_build_dependencies.sh
+   ```
+
+3. Create a build folder:
+   ```sh
+   mkdir build && cd build
+   ``` 
+
+4. To cross compile OpenVINO Runtime for RISC-V devices, run `cmake` with specified `CMAKE_TOOLCHAIN_FILE` and `RISCV_TOOLCHAIN_ROOT`:
+   ```sh
+   cmake .. \
+     -DCMAKE_BUILD_TYPE=Release \
+     -DCMAKE_INSTALL_PREFIX=<openvino_install_path> \
+     -DENABLE_INTEL_CPU=ON \
+     -DENABLE_INTEL_GPU=OFF \
+     -DENABLE_INTEL_NPU=OFF \
+     -DENABLE_AUTO=ON \
+     -DENABLE_HETERO=ON \
+     -DENABLE_MULTI=ON \
+     -DENABLE_JS=OFF \
+     -DCMAKE_TOOLCHAIN_FILE=../cmake/toolchains/<toolchain_file> \
+     -DRISCV_TOOLCHAIN_ROOT=<xuantie_install_path> \
+     -DENABLE_TESTS=ON \
+     -DENABLE_SAMPLES=ON
+   ```
+   > **NOTE**: To build OpenVINO Runtime for different versions of RVV, you just need to specify corresponding toolchain files. For exmaple, you can replace `<toolchain_file>` with `riscv64-071-thead-gnu.toolchain.cmake` for RVV 0.7.1 and `riscv64-100-thead-gnu.toolchain.cmake` for RVV 1.0 respectively.
+
+   Then run `make` to build the project:
+   ```sh
+   make install -j$(nproc)
+   ```
+
+### Build without RVV
+0. Prerequisite - build `riscv-gnu-toolchain`:
+   ```sh
+   git clone https://github.com/riscv-collab/riscv-gnu-toolchain.git
+   cd riscv-gnu-toolchain
+   ./configure --prefix=/opt/riscv
+   make linux build-qemu -j$(nproc)
+   ```
+   > **NOTE**: The `build-qemu` target is optional here as well. More information can be seen [here](https://github.com/riscv-collab/riscv-gnu-toolchain).
+
+1. Clone OpenVINO repository and init submodules:
+   ```sh
+   git clone https://github.com/openvinotoolkit/openvino.git
+   cd openvino
+   git submodule update --init --recursive
+   ```
```

Comment:
looks like we have copy paste with the previous toolchain approach. 
Maybe we can separate build steps on 2 stages: build toolchain(s) and then build OV with specific toolchain. Last step will be mentioned only once in this document.

Reply:
What you suggested does have a point! I will reorganize this part!